app-games/src/main/java/com/aptoide/android/aptoidegames/analytics/FirebaseAnalyticsSender.kt

```diff
@@ -11,11 +13,22 @@ class FirebaseAnalyticsSender @Inject constructor(
   private val firebaseAnalytics: FirebaseAnalytics,
 ) : AnalyticsSender {
   override fun setUserProperties(vararg props: Pair<String, Any?>) = props.forEach {
+    if (BuildConfig.DEBUG) {
+      Timber.tag("GA").i("set UP  ${it.first} = ${it.second}")
```

Comment:
Wouldn't it be easier to just use Timber debug logs (`Timber.d(...)`)?

Reply:
it is `i` to stay on the info level.
And we need a tag just to distinguish analytics output from any other.